### ISSUE TITLE
A11y: "Settings" link in login view lacks accessible name when not hovered

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/browser#1989]
## Describe the Bug

The cog/settings link on the login view hides the "Settings" text using `visibility:hidden`. This also hides the text from assistive technologies, resulting in the link being announced as "slash environment" (as a fallback, screen readers will announce the link's href) only.

## Steps To Reproduce

1. Using a screen reader, open the popup (when not logged in)
2. Set focus on the cog icon/settings link (using `Tab` until the link is reached)

## Expected Result

* the link should be announced as "Settings"

## Actual Result

* the link is announced as "slash environment" (falling back to the `href`)

## Screenshots or Videos

![The cog/settings link when focused, but not hovered](https://user-images.githubusercontent.com/895831/127752316-30ebbc06-ecfb-4df2-bfd8-9d8b3cb9195f.png)

## Environment

- Operating system: Windows 10
- Browser: Chrome 92
- Build Version (go to "Settings" → "About" in the app): 1.51.1

## Additional Context

In addition, it would be nice to visually show the "Settings" text when the link is focused, not just when hovered with the mouse
